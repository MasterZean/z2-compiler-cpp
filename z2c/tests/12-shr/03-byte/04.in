Byte{ 0b00000001} >> 1
Byte{ 0b00000010} >> 1
Byte{ 0b00000100} >> 1
Byte{ 0b00001000} >> 1
Byte{ 0b00010000} >> 1
Byte{ 0b00100000} >> 1
Byte{ 0b01000000} >> 1
Byte{ 0b10000000} >> 1
Byte{-0b00000001} >> 1
Byte{-0b00000010} >> 1
Byte{-0b00000100} >> 1
Byte{-0b00001000} >> 1
Byte{-0b00010000} >> 1
Byte{-0b00100000} >> 1
Byte{-0b01000000} >> 1
Byte{-0b10000000} >> 1
