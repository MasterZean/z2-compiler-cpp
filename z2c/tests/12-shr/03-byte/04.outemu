Byte{0}
Byte{1}
Byte{2}
Byte{4}
Byte{8}
Byte{16}
Byte{32}
Byte{64}
Byte{255}
Byte{255}
Byte{254}
Byte{252}
Byte{248}
Byte{240}
Byte{224}
Byte{192}
