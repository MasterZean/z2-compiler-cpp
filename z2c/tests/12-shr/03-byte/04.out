(uint8)(1) >> 1;
(uint8)(2) >> 1;
(uint8)(4) >> 1;
(uint8)(8) >> 1;
(uint8)(16) >> 1;
(uint8)(32) >> 1;
(uint8)(64) >> 1;
(uint8)(128) >> 1;
(uint8)(-1) >> 1;
(uint8)(-2) >> 1;
(uint8)(-4) >> 1;
(uint8)(-8) >> 1;
(uint8)(-16) >> 1;
(uint8)(-32) >> 1;
(uint8)(-64) >> 1;
(uint8)(-128) >> 1;
